CMakeLists.txt

```diff
@@ -870,6 +874,11 @@ if (Seastar_SSTRING)
     PUBLIC SEASTAR_SSTRING)
 endif ()
 
+if (SEASTAR_HAVE_EXECINFO)
+target_compile_definitions (seastar                                      
+	PUBLIC SEASTAR_HAVE_EXECINFO)
```

Comment:
could you use two spaces for indent here?

Reply:
changed